lib/charms/observability_libs/v0/juju_topology.py

```diff
@@ -0,0 +1,306 @@
+# Copyright 2022 Canonical Ltd.
```

Comment:
why did you add juju_topology?

Reply:
It's a dependency: the new prometheus_scrape (after fetch-lib) depends on it.